src/main/java/oliver/Ui.java

```diff
@@ -0,0 +1,165 @@
+package oliver;
+
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user
+ */
+public class Ui {
+    private Scanner sc = new Scanner(System.in);
+
+    /**
+     * Shows the welcome message to the user.
+     *
+     * @return string representation of the welcome message
+     */
+    public String showWelcome() {
+        return "Hello! I'm Oliver.\nWhat can I do for you?";
+    }
+
+    /**
+     * Shows the farewell message to the user.
+     *
+     * @return string representation of the farewell message
+     */
+    public String showBye() {
+        return "Bye. Hope to see you again soon!";
+    }
+
+    /**
+     * Reads the input from the user.
+     *
+     * @return string representation of user's input
+     */
+    public String readInput() {
+        return this.sc.nextLine();
+    }
+
+    /**
+     * Closes the scanner.
+     */
+    public void close() {
+        this.sc.close();
+    }
+
+    /**
+     * Shows a confirmation message when a task is added to the list.
+     *
+     * @param addedTask the task that has been added
+     * @param numTasks the number of tasks in the list
+     * @return string representation of the confirmation message
+     */
+    public String showAdd(Task addedTask, int numTasks) {
```

Comment:
Could this method name be more descriptive? eg. showAdded, since it shows that the task is added.